contrib/kafka/pkg/reconciler/resources/receive_adapter.go

```diff
@@ -80,6 +81,33 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) *v1.Deployment {
 			},
 		})
 	}
+	RequestResourceCPU, err := resource.ParseQuantity(args.Source.Spec.Resources.Requests.ResourceCPU)
+	if err != nil {
+		RequestResourceCPU = resource.MustParse("100m")
```

Comment:
btw. must be less than or equal to cpu limit

Reply:
see: https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit